wearApp/src/main/java/dev/johnoreilly/confetti/wear/components/PlaceholderButton.kt

```diff
@@ -0,0 +1,107 @@
+package dev.johnoreilly.confetti.wear.components;
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.unit.dp
+import androidx.wear.compose.foundation.ActiveFocusListener
+import androidx.wear.compose.material3.Button
+import androidx.wear.compose.material3.ButtonColors
+import androidx.wear.compose.material3.ButtonDefaults
+import androidx.wear.compose.material3.PlaceholderDefaults
+import androidx.wear.compose.material3.PlaceholderState
+import androidx.wear.compose.material3.placeholder
+import androidx.wear.compose.material3.placeholderShimmer
+import androidx.wear.compose.material3.rememberPlaceholderState
+import kotlinx.coroutines.launch
+
+/**
+ * A placeholder chip to be displayed while the contents of the [Chip] is being loaded.
+ */
+@Composable
+public fun PlaceholderButton(
+    modifier: Modifier = Modifier,
+    onClick: () -> Unit = {},
+    placeholderState: PlaceholderState = rememberActivePlaceholderState { false },
+    secondaryLabel: Boolean = true,
+    icon: Boolean = true,
```

Comment:
if this is a Boolean param expressing if it has an icon or not, perhaps `hasIcon` would read better?